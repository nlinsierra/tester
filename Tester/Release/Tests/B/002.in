6H 9D JH 2C TD 8D 9S 2D 4D JC 5D KS 4H 8S TC QD JS 3S 7H 6C 3C AD 4S 5C 5S 4C 7D QS 5H 8C 3H 2S
