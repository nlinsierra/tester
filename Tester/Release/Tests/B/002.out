20
7C 9C QC KC AC 3D 6D JD KD 2H 8H 9H TH QH KH AH 6S 7S TS AS
